### ISSUE TITLE
camvid palette

### ISSUE THREAD

**Issue by endaoguansanlu** [ShengtianSang/AFMA#5]
Excellent work! Hello, I am studying your paper recently, can you provide the palette of the camvid dataset? I want to know if the colors (rgb) for the 11 categories could match, looking forward to your reply!

**Comment by ShengtianSang** [comment]
Hi, thanks for your interest in my work. Below is the Camvid palette. I'm unsure if I changed this part of the code. Please contact me if any of the colors don't correspond. Thank you.

def formatting_label_camvid(mask):
    palette = {
        0: (102, 216, 249),  # sky
        1: (115, 109, 60),  # building
        2: (192, 192, 128),  # column_pole
        3: (125, 125, 125),  # road
        4: (0, 0, 192),  # sidewalk
        5: (30, 154, 12),  # Tree
        6: (249, 219, 47),  # SignSymbol
        7: (64, 64, 128),  # Fence
        8: (64, 0, 128),  # Car
        9: (255, 26, 26),  # Pedestrian
        10: (0, 128, 192),  # Bicyclist
        11: (236, 236, 236)  # Void
    }

**Comment by endaoguansanlu** [comment]
Thank you so much for your prompt reply! In addition, I found that the official download link of the CamVid dataset seems to be invalid. Could you please provide the complete CamVid training set and test set through Baidu cloud or google drive? Looking forward to your reply!

**Comment by ShengtianSang** [comment]
This link works for me (http://mi.eng.cam.ac.uk/research/projects/VideoRec/CamVid/). Hope it could help you to get the dataset.
